gpiozero/boards.py

```diff
@@ -102,6 +69,61 @@ def toggle(self):
         for led in self.leds:
             led.toggle()
 
+
+
+class LEDBoard(LEDCollection):
+    """
+    Extends :class:`CompositeDevice` and represents a generic LED board or
+    collection of LEDs.
+
+    The following example turns on all the LEDs on a board containing 5 LEDs
+    attached to GPIO pins 2 through 6::
+
+        from gpiozero import LEDBoard
+
+        leds = LEDBoard(2, 3, 4, 5, 6)
+        leds.on()
+
+    :param int \*pins:
+        Specify the GPIO pins that the LEDs of the board are attached to. You
+        can designate as many pins as necessary.
+
+    :param bool pwm:
+        If ``True``, construct :class:`PWMLED` instances for each pin. If
+        ``False`` (the default), construct regular :class:`LED` instances. This
+        parameter can only be specified as a keyword parameter.
+    """
+
+    def close(self):
+        self._stop_blink()
+        super(LEDBoard, self).close()
+
+    @property
+    def value(self):
+        """
+        A tuple containing a value for each LED on the board. This property can
+        also be set to update the state of all LEDs on the board.
+        """
+        return tuple(led.value for led in self._leds)
+
+    @value.setter
+    def value(self, value):
+        self._stop_blink()
+        for l, v in zip(self.leds, value):
+            l.value = v
+
+    def on(self):
+        self._stop_blink()
+        super(LEDBoard, self).on()
+
+    def off(self):
+        self._stop_blink()
+        super(LEDBoard, self).off()
+
+    def toggle(self):
+        self._stop_blink()
+        super(LEDBoard, self).toggle()
+
     def blink(
```

Comment:
Right now there isn't a reason ... but I didn't want to include blink because in future I do intend to add PWM to the LEDBarGraph class. Right now, both blink states (off and on) have a reasonable representation in the value of the bar graph (0 and 1). However, consider a blink which supports fading (with PWM) - if all the LEDs are on at half strength what value on the chart is that? Basically blink, once PWM is added, will utilise states that aren't valid in the graph so I figured it's better to leave it in the board class and just say "blinking's not a valid use-case for a graph component" (which doesn't seem too unreasonable to me).